presentation/src/main/java/com/anytypeio/anytype/presentation/sets/ViewerLayoutWidgetUi.kt

```diff
@@ -0,0 +1,108 @@
+package com.anytypeio.anytype.presentation.sets
+
+import com.anytypeio.anytype.core_models.Block
+import com.anytypeio.anytype.core_models.DVViewer
+import com.anytypeio.anytype.core_models.DVViewerType
+import com.anytypeio.anytype.core_models.Id
+import com.anytypeio.anytype.core_models.Relations
+import com.anytypeio.anytype.domain.objects.StoreOfRelations
+
+data class ViewerLayoutWidgetUi(
+    val viewer: Id?,
```

Comment:
is it a valid state when viewer id is null? if it is not, then `Id` should not be nullable here.

Reply:
In the case when the DataView's View has not arrived yet, either null or "" should be set here.